guide/14-deep-learning/faster-rcnn-object-detector.ipynb

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
<p>Region Proposal Network, like other region proposal algorithms, inputs an image and returns regions of interest that contain objects. An RPN also returns an "objectness" score <s>which</s> <em>that</em> measures how likely the region is<s> supposed</s> to have an object vs. a background [1]. In Faster R-CNN, the RPN and the detect network share the same backbone. The last shared layer of the backbone provides a feature map of the image <s>which</s> <em>that</em> is used by the RPN to propose regions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/897/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>